docs/source/user_guide/installation.rst

```diff
@@ -0,0 +1,77 @@
+============
+Installation
+============
+
+Updating Dependencies
+=====================
+
+Installing from PyPI
+--------------------
+
+When installing ``janus-core``, dependencies are automatically selected to be consistent with all
+``extras``.
+
+For an individual MLIP, you may be able to upgrade dependencies beyond these defaults,
+to include newer features or bug fixes. For example, to upgrade PyTorch to the latest version:
+
+.. code-block:: bash
+
+    python3 -m pip install -U torch
+
+
+or a specific version:
+
+.. code-block:: bash
+
+    python3 -m pip install torch==2.5.1
```

Comment:
```suggestion
    python3 -m pip install -U torch==2.5.1
```

Reply:
Does this actually change the behaviour? I know it'll change the `torch` version either way, but I suppose sub-dependencies may not update to their highest compatible version otherwise?

Reply:
good point... in general I do it without the version... 
some deps may be updated depends of the package constraints, if you want to update deps too --upgrade-strategy eager